common/web/utils/src/test/managedPromise.js

```diff
@@ -1,6 +1,6 @@
 import { assert } from 'chai';
 
-import ManagedPromise from '../../build/obj/managedPromise.js';
+import { ManagedPromise } from '@keymanapp/web-utils';
```

Comment:
Hmm, is this self-referential? Is that a good thing?

Reply:
`src/test/...`:  we're importing the package for its unit test.